cloudslang-cli/src/main/java/io/cloudslang/lang/cli/utils/MetadataHelperImpl.java

```diff
@@ -63,8 +69,28 @@ private Metadata cleanWhiteSpaces(Metadata metadata, String lineBreak) {
     }
 
     private void cleanMapWhiteSpaces(Map<String, String> map, String lineBreak) {
-        for (String key : map.keySet()) {
-            map.put(key, map.get(key).trim().replaceAll("[\\s]+" + lineBreak, lineBreak));
+        for (Map.Entry<String, String> entry : map.entrySet()) {
+            map.put(entry.getKey(), entry.getValue().trim().replaceAll("[\\s]+" + lineBreak, lineBreak));
+        }
+    }
+
+    private static class CustomOrderMetadataRepresenter extends Representer {
+        @Override
+        protected Set<Property> getProperties(Class<? extends Object> type)
+                throws IntrospectionException {
+            Set<Property> result = new LinkedHashSet<>();
+            Set<Property> set = super.getProperties(type);
+            for (Property property : set) {
+                if (property.getType().equals(String.class)) {
+                    result.add(property);
+                }
+            }
+            for (Property property : set) {
```

Comment:
All the `String`'s are being added to the result, only after that all the `Map`'s are added to the result. The properties are ordered by name(prerequisites is after outputs). The Metadata class only contains `String` and `Map` properties(this is a custom order specific to the Metadata class). A new method can be extracted for those two `for` loops.